keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -431,7 +431,8 @@ def compile(
         rpn_box_loss = _validate_and_get_loss(rpn_box_loss, "rpn_box_loss")
         if rpn_classification_loss == "BinaryCrossentropy":
             rpn_classification_loss = tf.keras.losses.BinaryCrossentropy(
-                from_logits=True, reduction=tf.keras.losses.Reduction.SUM
+                from_logits=True,
+                reduction=tf.keras.losses.Reduction.SUM_OVER_BATCH_SIZE,
```

Comment:
This actually changes the scaling needed for learning rate to rely on batch size.  This is a good reason to use sum instead of `SUM_OVER_BATCH_SIZE`.  

If you use SUM and you use learning rate 0.01, it doesn't matter if batch size is 256 or 8 or 1, you'll move by 0.01*sum(dL_i/dW) for all i in the training set.  The same is not the case for sum_over_batch size, which actually reduces the loss scale and therefore requires adjustment of learning rate.